### DISCUSSION TITLE
Dropdown menu stuck open all the time

### DISCUSSION THREAD

**Discussion by markkimsal** [community/community#61561] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

The (I don't know what to call it) "New Repository" menu is always open for me.  This started maybe 6 weeks ago?  It's something with CSS @supports selector(:open) and display: revert.  It's very very annoying and blocks some functionality unless I F12 and remove the node from every page I visit or disable the display:revert setting  :(

Does this happen for anyone else?

![image](https://github.com/community/community/assets/54099/13088112-5805-41db-bd1b-99fe8865d061)

**Comment by cx-shahar-septon** [comment]
This issue is local to your machine, probably the onClick event is not handled correctly.
I would advise to either:
1. Hard-refresh the page, or,
2. Change your browser
